Consul/Client.cs

```diff
@@ -148,6 +151,12 @@ public ConsulClientConfiguration()
             UriBuilder consulAddress = new UriBuilder("http://127.0.0.1:8500");
             ConfigureFromEnvironment(consulAddress);
             Address = consulAddress.Uri;
+
+            string ns = Environment.GetEnvironmentVariable("CONSUL_HTTP_TOKEN");
```

Comment:
```suggestion
            string ns = Environment.GetEnvironmentVariable("HTTPNamespaceEnvName");
```

Reply:
My mistake. But that variable is incorrect as well. It’s supposed to be `CONSUL_HTTP_NAMESPACE` I think. I’ll take a look again.

Reply:
Yeah, it's `CONSUL_NAMESPACE` https://github.com/hashicorp/consul/blob/main/api/api.go#L88